app/views/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hello Rails React</title>
+</head>
+<body>
+  <div id="root"></div>
+</body>
+</html>
```

Comment:
- [ ] While I will encourage you to add this file to a folder then use the route file to direct it, you also don't need all the doctype and co, you only need the `<div id="root"></div>` because all other setup is in the layouts/application.html.erb file already.

Reply:
Suggested changes have been implemented.